maven-embedder/src/main/java/org/apache/maven/cli/logging/Slf4jConfigurationFactory.java

```diff
@@ -65,8 +67,14 @@ public static Slf4jConfiguration getConfiguration(ILoggerFactory loggerFactory)
             }
         } catch (IOException | ClassNotFoundException | IllegalAccessException | InstantiationException e) {
             e.printStackTrace();
+            hadException = true;
         }
 
-        return new UnsupportedSlf4jBindingConfiguration(slf4jBinding, supported);
+        return new UnsupportedSlf4jBindingConfiguration(
+                slf4jBinding,
+                supported,
```

Comment:
warning seems the right level error. If someone is treating this as an error, that's their choice and problem, not ours.

Reply:
Not really, maven is wrong assuming it knows the entire ecosystem, it is perfectly valid to use a slf4j binding maven does not know - or if you are right it means we should drop slf4j from maven because we don't use the abstraction at the end.
If it is right then we must not issue a warning (an error) when it is an expected case  - the use case triggering this change.
Guess it was more a default implementation than an intent to say a custom binding is intended.

Assuming any custom binding will bring a custom maven integration is also a high assumption for the little usage we do of that integration (a single CLI toggle which is rarely used in practise) so hope we don't go that path which would be the alternative.

Reply:
If this is valid, we shouldn't log at all. If it isn't, log the warning. An INFO level log isn't right here.

Reply:
I rewrote the PR to not play with levels, please check the new proposal to solve this issue